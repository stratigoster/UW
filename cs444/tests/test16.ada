PACKAGE foo IS
  type Week is (Sun,Mon,Tue,Wed,Thu,Fri,Sat);
  type WeekEnd is (Sat,Sun);
  myFaveDay : Week := Sun; -- overloading works!
  i : INTEGER := 12 * 2 * myFaveDay;
--                    ^^^^^^^^^^^^^
-- enumeration elements are not integers! (Ada is not C)
END foo;
