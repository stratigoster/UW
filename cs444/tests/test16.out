***ERROR***:5:25:	MUL  works on numeric values of the same type

------
Compilation error
------
